doc/api/tls.md

```diff
@@ -789,8 +790,6 @@ added: v0.11.3
 * `port` {number}
 * `host` {string}
 * `options` {Object}
-  * `host` {string} Host the client should connect to.
```

Comment:
@Qard So ... what's the behaviour? Or should that be a separate PR, and if so, should I remove this change for now?